src/components/Class/ClassesList/styles.scss

```diff
@@ -0,0 +1,110 @@
+
+  
+
+.createClassButton{
```

Comment:
css naming convention usually goes like `create-class-button`.  What do you think if we use this kind of class names in css files? 🤔

Reply:
Done